src/StakingNode.sol

```diff
@@ -151,6 +151,10 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         unverifiedStakedETH = initialUnverifiedStakedETH;
     }
 
+    function initializeV3() external onlyStakingNodesManager reinitializer(3) {
+        // TODO - Should we initialize if there is nothing to set? only new version?
```

Comment:
you don't need to bump this if there's nothing to migrate or set in the storage with the new version